cmd/publishing-bot/config/rules.go

```diff
@@ -118,6 +120,7 @@ func readFromUrl(u *url.URL) ([]byte, error) {
 }
 
 func validateRepoOrder(rules *RepositoryRules) (errs []error) {
+	fmt.Printf("validating repository order\n")
```

Comment:
we use glog elsewhere if I am not mistaken.

Reply:
ack i can switch it over

Reply:
Done